frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -377,6 +377,13 @@ export function PollWorkerScreen({
     appPrecinct
   );
 
+  // Disable the audiotrack when on poll-worker screen
```

Comment:
did you mean to include this?

Reply:
yes I did. I don't think it's useful in Mississippi at all and it was a contributing factor to things sounding bad. I don't want to give the TTS system a workout if unnecessary.